chocolatey/plugins/module_utils/Packages.psm1

```diff
@@ -364,6 +386,10 @@ function ConvertTo-ChocolateyArgument {
     if ($AllowMultiple) { "--allow-multiple" }
     if ($AllowPrerelease) { "--prerelease" }
     if ($Architecture -eq "x86") { "--x86" }
+    if ($Checksum) { '--checksum', $Checksum }
+    if ($Checksum64) { '--checksum64', $Checksum64 }
+    if ($ChecksumType) { '--checksumtype', $ChecksumType }
+    if ($ChecksumType64) { '--checksumtype64', $ChecksumType64 }
```

Comment:
```suggestion
    if ($Checksum) { 
        '--checksum', $Checksum
        if ($ChecksumType) { '--checksumtype', $ChecksumType }
    }
    if ($Checksum64) {
        '--checksum64', $Checksum64
        if ($ChecksumType64) { '--checksumtype64', $ChecksumType64 }
    }
```

Is there any possibility of issues coming from overriding the checksum type without overriding the checksum value itself?

Given there's a default for the checksum type, there will _always_ be a value for that coming through from the module (iirc).

My suggested change here only changes the checksum type if the matching checksum value has been provided.

Not married to this change, feel free to disregard, it's just a thought that came up while reading through the changes.

Reply:
mmm oh, that's a good point... yeah there just probably shouldn't be a default for these at all. Choco has its own defaults anyway, and I think we're better off deferring to those instead, in case for some reason they change in future, less for us to mess with at that point.

Reply:
Removed the defaults here instead as they're not really needed. :3